### PR TITLE
Vaccine manuscripts proofreading

### PR DESCRIPTION
**Pull request by agitter**
I've started my proofreading of the vaccines manuscripts but didn't finish.  I'll finish at least the main text of both by Friday.

I changed the geopandas citations based on their [suggestion](https://geopandas.org/en/stable/about/citing.html) and used the version 0.10.2 DOI.

### CODE REVIEW

**Comment by agitter** [comment]
I'm done editing and commenting on the main text of the two vaccines manuscripts.  #1167 may already address some of my comments about the template variables so we could finish that first and merge it into this branch.

Will we submit the vaccines appendix to the journal with one of these manuscripts?  If we plan to submit it, I would also like to read through it in full (quickly) before submission, which I haven't done yet.

File: content/22.vaccines-trad.md
Comment (rando2) [inline content/22.vaccines-trad.md:431]: @agitter Did you review a version you built or did you review the one that is currently being built here? Because I don't think the `output` branch has updated since #1150 was merged in.

**Review by rando2 — COMMENTED** [review]
First pass of responses to traditional vaccines
I have no childcare due to COVID-19 again, so I apologize in advance!

File: content/22.vaccines-trad.md
Comment (rando2) [reply content/22.vaccines-trad.md:199]: https://github.com/agitter/covid19-review/pull/2

File: content/22.vaccines-trad.md
Comment (rando2) [inline content/22.vaccines-trad.md:366]: I can work on integrating this text back into the general discussion of the vaccine development. I think the idea was that it would be helpful to see the efficacy data all in one place to get a sense of the general success of the approach, but I do understand why it reads weirdly.

File: content/22.vaccines-trad.md
Comment (RLordan) [reply content/22.vaccines-trad.md:159]: @rando2 @agitter I agree with the response above, but also there is thought to be less of an immune response and reduced efficacy with inactivated vaccines. Adjuvants and boosters are often required.

File: content/22.vaccines-trad.md
Comment (RLordan) [reply content/22.vaccines-trad.md:254]: Yes with specific antibodies to a pathogen intended for treatment.

File: content/22.vaccines-trad.md
Comment (agitter) [reply content/22.vaccines-trad.md:182]: The table is referenced there or just the figure?  That could be a good place to introduce both, but it looked like only the figure was linked there.

File: content/22.vaccines-trad.md
Comment (agitter) [reply content/22.vaccines-trad.md:366]: After reading the novel vaccines manuscript, I understood the organization here better. But it didn't seem to match the earlier platforms in this manuscript very well.

File: content/22.vaccines-trad.md
Comment (agitter) [reply content/22.vaccines-trad.md:431]: I was reading through the July 26 GitHub pages version.  There have been a lot of changes since then, so I expect this has been resolved.  I wanted to make a new, merge in all your updates, and then remove this comment after confirming the figure was fixed.

File: content/22.vaccines-trad.md
Comment (rando2) [reply content/22.vaccines-trad.md:159]: Thank you @RLordan! I updated it.

File: content/22.vaccines-trad.md
Comment (rando2) [inline content/22.vaccines-trad.md:172]:
```suggestion
Several whole-virus vaccines have been developed against COVID-19 and are available in countries around the world (Table @tbl:approved-whole-virus).
```

File: content/22.vaccines-trad.md
Comment (rando2) [reply content/22.vaccines-trad.md:182]: You're absolutely right, I can't read. I just suggested adding it to line 72 above.

**Review by rando2 — APPROVED** [review]
These edits look great. @agitter if my suggestions on 22 look alright, would you want to commit them and merge? I will address the inline comments in a separate pr

**Comment by agitter** [comment]
I'm close to pushing my very minor edits to the appendix, so let's include that in this pull request before merging.

**Comment by agitter** [comment]
I finished reading through the appendix and pushed my changes.  With these edits, I also approve submitting after @rando2 finishes her final edits.  I'm going to be traveling and won't be online much August 13-21 to review those changes.

I suggest we keep the appendix online only and don't submit it to the journal.  Parts of it are incomplete and not written as thoroughly as the main text.  Parts are well written but are from early 2021 and do not contribute much for a journal reader in late 2022.  When we refer to the appendix in the text, we can link to https://greenelab.github.io/covid19-review/ or something similar.

**Comment by rando2** [comment]
Thank you so much @agitter! I can easily remove the references to the Appendix in the text. Safe travels!

File: content/24.vaccines-app.md
Comment (rando2) [inline content/24.vaccines-app.md:2]: Done-ish in a new PR that I'll open as soon as I incorporate these edits!

File: content/24.vaccines-app.md
Comment (rando2) [inline content/24.vaccines-app.md:407]: This is Novavax, here is the original PR: #1063

File: content/24.vaccines-app.md
Comment (agitter) [reply content/24.vaccines-app.md:2]: Great! Yes, let's save that for a separate pull request.

File: content/24.vaccines-app.md
Comment (agitter) [reply content/24.vaccines-app.md]: I updated my comment in 65291c7. It's not trivial to keep these sentences about the phase III results because the previous paragraph says they aren't ready yet.  This pull request is huge, so if you're ready, let's merge it and stop making changes.

Comment (rando2) [reply content/24.vaccines-app.md]: That sounds good to me! I had updated the Novavax text this week but I'm worried it got lost in a merge conflict (resolving them amidst no-childcare-chaos is super difficult, it turns out.) It didn't take long to update, though, so I can fix it again if I can't figure out what happened!

**Comment by rando2** [comment]
@agitter I am going to make one more change here (splitting the appendix) if you're done editing. That will make it easier to compare against my changes to the appendix in #1171

**Comment by agitter** [comment]
I'm done editing here. Should should be able to push to this branch
image

But do whatever is easiest for you.

**Comment by rando2** [comment]
> I'm done editing here. Should should be able to push to this branch image
> 
> But do whatever is easiest for you.

Oh I didn't realize! That makes things a lot easier. I just pushed the text move here and will merge once it builds.

**Pull request merged**
